### PR TITLE
meta: update AUTHORS

### PR DESCRIPTION
**Pull request by aqrln**
Update AUTHORS using `tools/update-authors.js`.

### CODE REVIEW

File: AUTHORS [hunk 1]
```diff
@@ -4,3 +4,4 @@ Artem Chernenkiy <notthewhite@gmail.com>
 Alexey Orlenko <eaglexrlnk@gmail.com>
 Mykola Bilochub <nbelochub@gmail.com>
 Vlad Dziuba <dzyubavlad@gmail.com>
+lundibundi <shishugi@gmail.com>
```
Comment (aqrln) [inline AUTHORS]: @lundibundi I need your real name latinized ;)
It would also be nice if you set it in your Git config too.

Comment (lundibundi) [inline AUTHORS]: @aqrln Denys Otrishko

**Comment by aqrln** [comment]
ping @belochub

**Comment by aqrln** [comment]
Landed in 4e298cbb21e1815c28f730c6dbc272a243994cfc.

**Pull request merged**